package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "modbus-serial",
-  "version": "8.0.13",
+  "version": "8.0.14",
```

Comment:
The vertion bump is out of topic, can you undo this change ?

Reply:
Sure, rolled back.